libraries/lib-stretching-sequence/tests/AudioContainerHelper.h

```diff
@@ -0,0 +1,27 @@
+/*  SPDX-License-Identifier: GPL-2.0-or-later */
+/*!********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  TestWaveTrackMaker.cpp
+
+  Matthieu Hodgkinson
+
+**********************************************************************/
+#pragma once
+
+#include "AudioContainer.h"
+
+class AudioContainerHelper
+{
+public:
+   template <typename T = float>
+   static std::vector<T*>
+   GetData(const AudioContainer& container, size_t offset = 0u)
```

Comment:
You need `#include <cstddef>` to define `size_t` and fix Linux builds.

Reply:
Thank you!